recipes/newrelic/apm/dotNet/windows/net-install.yml

```diff
@@ -20,12 +20,18 @@ keywords:
 processMatch:
   - w3wp.exe
 
-validationNrql: "SELECT count(*) from Transaction WHERE host like '{{.HOSTNAME}}%' facet entityGuid since 10 minutes ago"
+validationNrql: "SELECT count(*) from Transaction WHERE host like '{{ .HOSTNAME }}%' facet entityGuid since 10 minutes ago"
 
 successLinkConfig:
   type: EXPLORER
   filter: '"`tags.language` = ''dotnet''"'
 
+inputVars:
+  - name: "IIS_ROOT_URLS"
```

Comment:
@jaffinito could you rename this input var with a NR_CLI_ prefix, this is a consistent prefix we use across all recipes

Reply:
Will do!